tests/cupy_tests/core_tests/test_ndarray_complex_ops.py

```diff
@@ -22,6 +22,20 @@ def test_conj_pass(self, xp, dtype):
         self.assertIs(x, y)
         return y
 
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_array_almost_equal()
+    def test_conjugate(self, xp, dtype):
+        x = testing.shaped_arange((2, 3), xp, dtype)
+        return x.conjugate()
+
+    @testing.for_all_dtypes(no_complex=True)
+    @testing.numpy_cupy_array_almost_equal()
+    def test_conjugate_pass(self, xp, dtype):
+        x = testing.shaped_arange((2, 3), xp, dtype)
+        y = x.conjugate()
+        self.assertIs(x, y)
+        return y
+
```

Comment:
Sorry, it seems I overlooked this test for whatever reason.
This test is exactly what I meant.
Please forget my comment... :bowing_man: